scientific_library/contrib/tvb/contrib/scripts/datatypes/connectivity.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
+
+from tvb.contrib.scripts.datatypes.base import BaseModel
+from tvb.datatypes.connectivity import Connectivity as TVBConnectivity
+
+
+class ConnectivityH5Field(object):
```

Comment:
Do we still need this class?

Reply:
This class is not used anymore. I will delete it.